src/openassetio-core/src/managerApi/ManagerInterface.cpp

```diff
@@ -74,6 +77,52 @@ void ManagerInterface::getWithRelationships(
     successCallback(idx, {});
   }
 }
+
+namespace {
+/*
+ * A dummy pager interface that acts as if it has no data. For use in
+ * getWithRelationship[s] default implementation.
+ */
+class IdentityEntityReferencePagerInterface : public managerApi::EntityReferencePagerInterface {
```

Comment:
[consider] Just as 'identity' is quite loaded in 3D graphics/maths. `EmptyEntityReferencePagerInterface`.

Reply:
Renamed as suggested in 90a04e42